contrib/vcl/source/vcl_interface.cc

```diff
@@ -0,0 +1,126 @@
+#include "contrib/vcl/source/vcl_interface.h"
+
+#include "source/common/network/address_impl.h"
+
+#include "contrib/vcl/source/vcl_io_handle.h"
+#include "vppcom.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Network {
+namespace Vcl {
+
+namespace {
+
+/**
+ * Max number of workers supported. Used in declaration of epoll_handles array
+ */
+const int MaxNumWorkers = 128;
```

Comment:
What happens if Envoy has concurrency > 128?

Reply:
It could be a vector with pre-allocated capacity of 128. Should be enough for foreseeable future not to trigger reallocation.

Reply:
Just pushed a patch wherein we read concurrency at bootstrap time and use it to reserve space in a vector (as opposed to array).